src/core/nem/external/nacl-fast.js

```diff
@@ -651,6 +651,22 @@
         return 0;
     }
 
+    function crypto_shared_key_hash(shared, pk, sk, hash) {
```

Comment:
shared_key => shared_secret? or perhaps just call it get_scalar_mult_result?

Reply:
I copied this from the old SDK.
All custom methods added `crypto_` as a prefix.

maybe call `crypto_scalar_mult_result` ?